swap_meet/item.py

```diff
@@ -1,2 +1,32 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else: 
+            self.id = id
+
+        self.condition = condition
+
+    def get_category(self):
+        return "Item"
+    
+    def condition_description(self):
+        descriptions = {
+            0: "Worst condition",
+            1: "Heavily used",
+            2: "Fair",
+            3: "Good",
+            4: "Like new",
+            5: "Best condition"
+        }
+
+        if self.condition in descriptions:
+            return descriptions[self.condition]
+        else:
+            return False
+
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
```

Comment:
Remember that we have a way to retrieve the category of our class instance. We could use that method within our return statement here:

```suggestion
        return f"An object of type {self.get_category()} with id {self.id}."
```

This would also make it possible to use this parent stringify method within our child classes!